app/src/main/java/org/dhis2/usescases/main/MainModule.kt

```diff
@@ -38,6 +41,21 @@ class MainModule(val view: MainView) {
         )
     }
 
+    @Provides
+    @PerActivity
+    fun provideHomeRepository(d2: D2, charts: Charts?): HomeRepository {
```

Comment:
Where does charts come from?

Reply:
It is injected from the serverComponent as usual :)